tests/src/sub/nesting/e2e.test.ts

```diff
@@ -32,6 +32,7 @@ describe('Composite nesting tests', () => {
     const collectionA = await helper.nft.mintCollection(alice, {permissions: {nesting: {tokenOwner: true}}});
     const collectionB = await helper.ft.mintCollection(alice);
     const collectionC = await helper.rft.mintCollection(alice);
+    const collectionD = helper.ft.getCollectionObject(0);
```

Comment:
Эта коллекция особенная, давай ее выделим

```suggestion
    const collectionNative = helper.ft.getCollectionObject(0);
```